### ISSUE TITLE
Kaiser Permanente: Azure chat bot improperly using aria-roledescription attribute - missing semantic role

### ISSUE THREAD

**Issue by Amit8527510735** [microsoft/BotFramework-WebChat#3592]
**Accessibility 161217: Kaiser Permanente:** Azure chat bot improperly using aria-roledescription attribute - missing semantic role

**Product name, version, and build:** Azure Chat Bot, Edge/Chrome v.86

**OS version and build:** Windows 10

**AT version and build (if applicable):** JAWS 2020, NVDA 2020.3

Does this repro with other AT? (If applicable) Yes, Narrator and NVDA

**Share access to the environment (if applicable).:** https://fdoorq02.azurefd.net/mock-aem-my-health

**Prerequisite (if any)** 

**Description**
When accessing the customer's chat experience, a number of elements are using the aria-roledescription property to add more detail to the semantic role. In several cases where this is used, the roledescription property MUST be accompanied by a semantic role (aria or otherwise). 
Accessible Rich Internet Applications (WAI-ARIA) 1.1 (w3.org)

**Repro steps**
1. Open URL to environment in Chrome/edge
2. Submit a zip code and press Enter
3. Right-click the box labeled "Great! Now, Let’s focus your search a little more."
4. Select inspect element
5. Observe in source code for chat bubble, there is a DIV given aria-roledescription="message"

<div aria-roledescription="message" class="webchat__stacked-layout__message-row" id="webchat__stacked-layout__id--gvl4q"><div class="webchat--css-dgans-111jw2m">Bot said:</div><div class="webchat__bubble webchat__bubble--hide-nub webchat--css-dgans-1qo0vqe webchat--css-dgans-8wa66o webchat__stacked-layout__message"><div class="webchat__bubble__nub-pad"></div><div class="webchat__bubble__content"><div class="markdown webchat--css-dgans-113chkw"><p><strong>Great! Now, Let’s focus your search a little more.</strong></p>
<p>What service type are you looking for in the 10000 area?</p>
</div></div></div></div>

**Screenshots N/A**

**Actual results**
The element given aria-roledescription needs to have a semantic role. A div given aria-roledescription is insufficient

**Expected Results**
User would understand that this is an interactive control with a semantic role. If not an interactive control, should not use aria-roledescription

**MAS Reference:**
[MAS 4.1.2 - Name, Role, Value](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7b248054a6-5e68-4771-9e1e-242fb5025730%7d)

![MAS4 1 2_Chatbot](https://user-images.githubusercontent.com/53263029/98231679-be121e00-1f82-11eb-83eb-ddbf6f91afae.png)

**Comment by Amit8527510735** [comment]
#A11yMAS;#Accessibility;#MAS4.1.2;#HCL;A11yeDAD;#HCL_BotFramework_WebChat;

**Comment by corinagum** [comment]
This is a duplicate of https://github.com/microsoft/BotFramework-WebChat/issues/3537, and is fixed in our R11 release, of which the CDN version is planned to be released today.
